pdr_backend/lake/gql_data_factory.py

```diff
@@ -59,23 +56,19 @@ def __init__(self, ppss: PPSS):
         self.record_config = {
             "fetch_functions": {
                 Prediction: fetch_filtered_predictions,
-                Subscription: fetch_filtered_subscriptions,
                 Trueval: fetch_truevals,
                 Payout: fetch_payouts,
-                Slot: fetch_slots,
             },
             "config": {
                 "contract_list": contract_list,
             },
             "gql_tables": [
                 dn.get_lake_table_name()  # type: ignore[attr-defined]
-                for dn in [Prediction, Subscription, Trueval, Payout, Slot]
+                for dn in [Prediction, Trueval, Payout]
```

Comment:
I would suggest adding a constant in lake_mapper called REGISTERED_LAKE_TYPES or something similar. It seems this list of types is a repeating thing. In all fairness, I did notice this while working on the ETL but I was lazy to do it myself :)

Reply:
I mean... we are looking inside the `self.record_config` param from the GQLDataFactory object... This gets used inside the ETL to create the lake.

I feel like it's pretty self-explanatory but I'll improve this so we only have to define it once, and the nomenclature is better.

Reply:
Done, please review.